src/main/java/bio/terra/app/controller/RepositoryApiController.java

```diff
@@ -178,10 +180,14 @@ private AuthenticatedUserRequest getAuthenticatedInfo() {
     public ResponseEntity<EnumerateDatasetModel> enumerateDatasets(
             @Valid @RequestParam(value = "offset", required = false, defaultValue = "0") Integer offset,
             @Valid @RequestParam(value = "limit", required = false, defaultValue = "10") Integer limit,
-            @Valid @RequestParam(value = "sort", required = false, defaultValue = "created_date") String sort,
-            @Valid @RequestParam(value = "direction", required = false, defaultValue = "asc") String direction,
+            @Valid @RequestParam(value = "sort",
+                required = false,
+                defaultValue = "CREATED_DATE") EnumerateSortByParam sort,
```

Comment:
I thought this was case-insensitive?

Reply:
Doh! This is leftover from before I made it case-insensitive. Good catch!

Reply:
Oh that makes sense!